lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -544,7 +573,7 @@ class PatternStopContents extends Component<Props, State> {
                 selectType='continuousDropOff'
                 shouldHaveDisabledOption
                 title='Indicates whether a rider can alight from the transit vehicle at any point along the vehicle\u2019s travel path.'
-                value={(patternStop.continuousDropOff || patternStop.continuousDropOff === 0) ? patternStop.continuousDropOff : 1}
+                value={numberOrDefault(patternStop.continuousDropOff, 1)}
```

Comment:
I'm also never seeing these fields rendered for flex or non-flex feeds

Reply:
You will have to show me the config you use. I see this:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/56846598/208186655-642f8817-a634-4341-9c49-0661f060838c.png">

Reply:
This is fine, now sure what was going on with my config here.